src/main/java/seedu/clinic/logic/commands/DeleteCommand.java

```diff
@@ -109,8 +120,13 @@ private CommandResult executeWarehouseRelatedDeletion(Model model) throws Comman
 
         Warehouse warehouseToUpdate = lastShownList.get(targetIndex.getZeroBased());
         try {
+            // Find the product with name matching the targetProductName
             Product matchedProduct = warehouseToUpdate.getProductByName(targetProductName.get());
+
+            // Remove the matchedProduct form the target warehouse
             Warehouse updatedWarehouse = warehouseToUpdate.removeProduct(matchedProduct);
+
+            // Update the warehouse list in the model
```

Comment:
Minor issue but perhaps these comments should start with "Finds", "Removes" and "Updates" instead!

Reply:
I'm not sure if the comment here needs to stick to this format but I guess I will just go with your suggestion, thanks!